opennms-config-model/src/main/resources/xsds/eventconf.xsd

```diff
@@ -70,6 +70,12 @@
           </annotation>
         </element>
 
+        <element maxOccurs="1" minOccurs="0" name="priority" type="int">
+          <annotation>
+            <documentation>The Priority of the Event Definition.</documentation>
```

Comment:
We should specify if priority 1 is low or high or what the range is that is allowed if not specified in a restriction.

Reply:
Let me know if the updated description is sufficient. Thanks.